block-layer/codecs/DAG-JSON.md

```diff
@@ -20,8 +30,8 @@ of `dag-json` can't use the native JSON parser and serializer.
 {"/": { "base64": String }}
 ```
 
-### Link Type
+#### Link Type
 
 ```javascript
 {"/": String /* base encoded CID */}
```

Comment:
If we want determinism, what's the required base encoding?

Reply:
That's my next PRs, I've already written it, I'll push it once we've merged this one :)